morse_code_decoder.rb

```diff
@@ -0,0 +1,21 @@
+def morse_code_decoder(cypher_string)
+  morse_hash = {
+    '.-' => 'a', '-...' => 'b', '-.-.' => 'c', '-..' => 'd', '.' => 'e', '..-.' => 'f',
+    '--.' => 'g', '....' => 'h', '..' => 'i', '.---' => 'j', '-.-' => 'k', '.-..' => 'l',
+    '--' => 'm', '-.' => 'n', '---' => 'o', '.--.' => 'p', '--.-' => 'q', '.-.' => 'r',
+    '...' => 's', '-' => 't', '..-' => 'u', '...-' => 'v', '.--' => 'w',
+    '-..-' => 'x', '-.--' => 'y', '--..' => 'z', '   ' => ' '
+  }
+
+  # decode cypher text
+  cypher_array = cypher_string.split(/\s/).map { |i| morse_hash[i] }
+  text = ''
+  cypher_array.each do |t|
+    text += (t || ' ')
+  end
+  text.split.join(' ').upcase
+end
```

Comment:
- [ ] As the instructions states, please create three reusable methods that does the folllowing

### First method
- it translates Morse code symbols into letters

### Second method
- it combines translated chars into a single word to be returned as the final output

### Third method
- it combines translated words into a whole sentence to be returned as the final output